packages/service/ni_measurement_plugin_sdk_service/measurement/service.py

```diff
@@ -189,9 +189,12 @@ def __init__(
         Args:
             service_config_path (Path): Path to the .serviceconfig file.
 
-            version (str): Version of the measurement service.
+            version (str): Version of the measurement service. Do not use this
```

Comment:
There is a mechanism we can use to tell clients not to use this argument: `warnings.warn("The MeasurementService 'version' parameter is deprecated. Instead, specify the 'version' field in the .serviceconfig file.", DeprecationWarning)`.

I don't know if I like emitting a warning that 100% of users will run into, but I guess that's what `PendingDeprecationWarning` is for.

Reply:
Yeah - I'm inclined not to warn for now - at least until we can come up with an automatic mechanism for bumping your version in the .serviceconfig file.  I agree that it's too noisy for too little (initial) benefit.

For .NET, my plan is to generate the `.serviceconfig` file from the build process via a set of msbuild properties/items.  I'm not sure if something similar for python exists?  If so, then it becomes compelling to put the version in the `.serviceconfig` file rather than the `measurement.py` file.  As things stand now, you have to manually update one or the other.

The only significant benefit (at this point) to the .sericeconfig file is that if you don't put the version in the .serviceconfig file, the Plug-in Library CLI will prevent you from publishing the service to a Plug-in Library.

Reply:
Ok, the current PR is fine.